### PR TITLE
IA-1454 IA-1250 automated welder hash update in reference.conf

### PR DESCRIPTION
**Pull request by Qi77Qi**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by Qi77Qi** [comment]
Verified that welder hash is updated properly when cluster gets restarted

**Review by akarukappadath — APPROVED** [review]
do we want an autotest for welder updates? Might be a long test since we would need to create a cluster, stop it, and restart it, but might be worth it to make sure that welder continues to be updated on users' clusters

**Comment by Qi77Qi** [comment]
@akarukappadath , good idea..will do

**Review by rtitle — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala [hunk 1]
```diff
@@ -218,6 +217,34 @@ class NotebookClusterMonitoringSpec extends GPAllocFixtureSpec with ParallelTest
         }
       }
     }
+
+    "should update welder on a cluster" in { billingProject =>
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala:221]: Nice!

**Review by akarukappadath — APPROVED** [review]
Awesome, thanks for taking care of this!

**Pull request merged**